pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -258,6 +262,16 @@
     // Name of load manager to use
     @FieldContext(dynamic = true)
     private String loadManagerClassName = "com.yahoo.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl";
+    @FieldContext(dynamic = true)
+    private String activeVersion = "";
```

Comment:
This feature only works with `ModularLoadManagerImpl` which has a centralized load manager (on the lead broker).

Sometimes, it may be desirable to rollback to a previous version and the same feature is desired (namely that each bundle is affected exactly once). 

Are you suggesting that the load manager parse the version strings in all the load reports and choose the one that looks the latest, and do away with `activeVersion` altogether? I'm open to that approach if everyone in the community uses dotted decimal version numbers that are comparable.